core/src/jvmMain/kotlin/com/powersync/PSJdbcSqliteDriver.kt

```diff
@@ -53,7 +53,7 @@ internal class PSJdbcSqliteDriver(
         listenersToNotify.forEach(Query.Listener::queryResultsChanged)
     }
 
-    private val connection: SQLiteConnection = DriverManager.getConnection(url, properties) as SQLiteConnection
+    val connection: SQLiteConnection = DriverManager.getConnection(url, properties) as SQLiteConnection
```

Comment:
Maybe I missed where this is used outside of this but why does this not need to be private anymore?

Reply:
We're now accessing the connection in `DatabaseDriverFactory.jvm.kt` to listen for updates on the inner database connection (`PSJdbcSqliteDriver`) and forward it to the `PsSqlDriver`.